### DISCUSSION TITLE
When is the release of 9.0.0 targeted for?

### DISCUSSION THREAD

**Discussion by walnutist** [ibis-project/ibis#8912] [Q&A] [answered]
I am planning our own release which depends on features added in 9.0.0. A rough idea/plan say which month would be just fine.
Thanks in advance!

**Comment by lostmygithubaccount** [comment, marked as answer]
hi @walnutist, it should be this week!

**Reply by walnutist** [reply]
Thanks for the reply. Love you guys' absolutely outstanding work, keep going!

**Reply by lostmygithubaccount** [reply]
or next week, apologies! still have some release blockers we're working through

you can install the prerelease versions (built weekly) in the meantime
